scalableminds/webknossos
# Show converting datasets

### URL of deployed dev instance (used for testing):
- https://showconversiondatasets.webknossos.xyz

### Steps to test:
- upload dataset that needs conversion
- go to dashboard
- => see blue alert box which shows the uploaded dataset
- => when upload is complete, alert box should be removed

### Issues:
- fixes #5482 

------
